actify/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "actify"
-version = "0.4.2"
+version = "1.0.0"
```

Comment:
Het is 0.5.0. Voor 1.0.0 kun je breaking changes in minor release nummers met semver zetten. 1.0.0 is de eerste stable releas